### ISSUE TITLE
Unexpected token "spread operator" of value "..." in Twig templates

### ISSUE THREAD

**Issue by dragondungeon1** [VincentLanglet/Twig-CS-Fixer#233]
### Rule(s) related to or rule(s) missing

The spread operator (...) is not recognized in Twig templates.

### Expected behavior

To spread out the properties of the attr object. However, it seems to be causing an error.

### Actual behavior

The spread operator is causing an error in several Twig templates in my project. The error message is "File is invalid: Unexpected token "spread operator" of value "..."".
The spread operator does work in the templates but the cs-fixer throws errors on the ... operator.

Here's an example of how I'm using the spread operator in these files:

```<twig:Button {{ ...attr }}>```

Any help on how to resolve this would be appreciated.

example output
```
 KO ./templates/crud/view.html.twig
 ------- ----------------------------------------------------------------------------------------------------------- 
  FATAL   35   |                                                                                                     
          36   |             <twig:Button size="lg" asLink href="{{ action.vars.href }}" {{ ...actionAttributes }}>  
          >>   | File is invalid: Unexpected token "spread operator" of value "...".                                 
          37   |                 {{ action.vars.label|raw }}                                                         
 ------- -----------------------------------------------------------------------------------------------------------
```

**Comment by VincentLanglet** [comment]
HI, if you look at other issues you'll see the same https://github.com/VincentLanglet/Twig-CS-Fixer/issues/230 which is solved in 2.10.0 https://github.com/VincentLanglet/Twig-CS-Fixer/releases/tag/2.10.0 do you use it ?

Do you have `symfony/ux-twig-component` installed in the same place than the fixer ?

**Comment by dragondungeon1** [comment]
@VincentLanglet yes symfony/ux-twig-component is installed in the same project as twig-cs-fixer.

updating to 2.10.0 did indeed the trick thanks for the support!
